### PR TITLE
High Temperature Gas-cooled Reactor quest

### PR DESCRIPTION
**Pull request by kuba6000**
image
image

### CODE REVIEW

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -317379,6 +317379,58 @@
...
+          "globalShare:1": 1,
+          "questLogic:8": "OR",
+          "repeat_relative:1": 1,
+          "name:8": "§6§lHigh Temperature Gas-cooled Reactor",
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "§6§lHTGR§r§6 is a ZPM tiered reactor meant to be used as a breeder reactor to make Lutetium and gasses used in void miners.\n\n§rEficiency is calculated based on how full the reactor is, overall it works EXPONENTIALLY better when full.\nThe formula for efficiency is (fuel/72000)^2\nIt has minimal amount of fuel needed to start operation (10%).\nNeeds 1A LuV and 4000L of coolant(depending on fuel) to operate.\nFor more info check descriptions of items, the structure can be checked by blueprint.\n\nInformation about Fuels:\n\n90% of fuel casing parts are recyclable\n\n§2Thorium fuel is turned into Lutetium with 1:1 ratio (but 90% efficiency) also multiplies coolant needed by 1x\n§aUranium fuel is turned into Krypton also multiplies coolant needed by 1.5x\n§cPlutionium fuel is turned into Xenon also multiplies coolant needed by 2x\n\n§Y§9You can find all items by typing htgr in nei."
```
Comment (GTNH-Afx237v7) [inline config/betterquesting/DefaultQuests.json:317411]: Efficiency (typo) / 4000L of coolant per pebble, per tick, per second?

Comment (kuba6000) [reply config/betterquesting/DefaultQuests.json:317411]: 4000L/s * (fuel multiplier) * (efficiency 1%-100%)

Comment (kuba6000) [reply config/betterquesting/DefaultQuests.json:317411]: Ok I specified it in the quest

**Comment by Dream-Master** [comment]
@kuba6000 reward is only hot tea ?

**Comment by kuba6000** [comment]
> @kuba6000 reward is only hot tea ?

Yes

**Comment by Dream-Master** [comment]
Did it need more casings? Its a multi i guess. Maybe add it to the quest. And make some nice lootbag /coin reward as well.

**Comment by kuba6000** [comment]
> Did it need more casings? Its a multi i guess. Maybe add it to the quest. And make some nice lootbag /coin reward as well.

Ok I have added coins as reward and casings as task.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**